### PR TITLE
[test] Selenium test for filter #200

### LINKED ISSUE #200: [test] Selenium test for Filters
Write selenium tests to ensure proper working of modified filters.
### Tasks

- [x] Log in to the admin site and ensure filters are visible on the change list page.
- [x] Test the filter dropdown is working properly.
- [x] Test filters are applied when the ``filter`` button is clicked.
- [ ] Test ``show all`` button works perfectly when clicked. **Not Req:** Removed this feature. 
- [x] Test ``clear all`` button working.

blocked by https://github.com/openwisp/openwisp-utils/issues/196.

### PR DESCRIPTION
**Pull request by niteshsinha17**
#### Changes
Written selenium tests for the filter.

#### Checklist

- [x] I have read the [contributing guidelines](http://openwisp.io/docs/developer/contributing.html#how-to-commit-your-changes-properly).
- [x] I have manually tested the proposed changes.
- [x] I have written new test cases to avoid regressions. (if necessary)
- [x] I have updated the documentation. (e.g. README.rst)
- [x] I have added [change!] to commit title to indicate a backward incompatible change. (if required)
- [x] I have checked the links added / modified in the documentation.
close #200

### CODE REVIEW

File: openwisp_utils/admin_theme/templates/admin/base_site.html [hunk 1]
```diff
@@ -14,6 +14,16 @@
     <link rel="stylesheet" type="text/css" href="{% static "admin/css/openwisp.css" %}" />
     <link rel="icon" type="image/x-icon" href="{% static "ui/openwisp/images/favicon.png" %}" />
 {% endif %}
+{% if OPENWISP_TEST_MODE %}
+    <link rel="stylesheet" type="text/css" href="{% static "admin/css/openwisp.css" %}" />
+    <link rel="icon" type="image/x-icon" href="{% static "ui/openwisp/images/favicon.png" %}" />
```
Comment (pniaps) [inline openwisp_utils/admin_theme/templates/admin/base_site.html:19]: I've not tested locally, but **openwisp.css** and **favicon.png** are duplicated while testing?

**Review by nemesisdesign — COMMENTED** [review]
Please rebase on the latest master.

**Review by atb00ker — COMMENTED** [review]
This one has too many dependencies to be merged, I'll review this when those are merged! :smile:

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
There's some conflicts with master. Could you fix those please?

**Review by nemesisdesign — APPROVED** [review]
Thanks :+1:

**Pull request merged**